lib/pact_broker/api/resources/pagination_methods.rb

```diff
@@ -4,9 +4,9 @@ module Resources
       module PaginationMethods
         def pagination_options
           {
-            page_number: request.query["pageNumber"]&.to_i,
-            page_size: request.query["pageSize"]&.to_i
-          }.compact
+            page_number: request.query["pageNumber"]&.to_i || 1,
```

Comment:
This should use the implementation that was deleted from the other method.

```
def pagination_options
          if request.query["pageNumber"] || request.query["pageSize"]
            {
              page_number: request.query["pageNumber"]&.to_i || 1,
              page_size: request.query["pageSize"]&.to_i || 100
            }
          else
            nil
          end
        end
```

The code you have there will always return pagination options, which is not backwards compatible. The code from the snippet above only returns pagination options if either a pageNumber or pageSize has been specified.